pkg/manager/config.go

```diff
@@ -11,17 +13,28 @@ import (
 // NewConfig is used to create a new configuration object with default values.
 // Values can be overridden by passing `managercfg.Opt` options.
 func NewConfig(opts ...managercfg.Opt) (managercfg.Config, error) {
```

Comment:
Maybe update the description or/and name of this function to indicate that it relies on CLI flags to make it obvious. WDYT?

Reply:
Added a note to the function godoc.